.github/CODEOWNERS

```diff
@@ -13,7 +13,7 @@
 #
 
 # Global owners, will be the owners for everything in the repo.
-*  @austinlparker @mtwo @flands @jparsana @tedsuo @open-telemetry/docs-approvers
```

Comment:
@austinlparker: All those users being removed are members of [docs-approvers](https://github.com/orgs/open-telemetry/teams/docs-approvers/members), except for @tedsuo. Shall I add @tedsuo to `docs-approvers`?

Reply:
yes, please